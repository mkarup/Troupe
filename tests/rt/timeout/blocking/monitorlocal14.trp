let val secret = true raisedTo `{secret}`
    val p = spawn (fn () =>
                      receive [ hn _ => () ];
                      if secret then
                        ()
                      else
                        _blockThread ()
                  )
    val _ = monitorlocal (p, `{}`)
in
  send (p, 42);
  receive [ hn m => m ]
end
