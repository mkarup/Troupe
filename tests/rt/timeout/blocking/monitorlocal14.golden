2024-12-11T16:27:06.710Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
